google/logging/v2/logging_gapic.yaml

```diff
@@ -82,6 +89,9 @@ interfaces:
       - parameters:
         - sink_name
         - sink
+    required_fields:
+      - sink_name
+      - sink
```

Comment:
(I can't seem to leave a comment on unmodified code) I think DeleteSink's parameter sink_name is required.